envoy.dependency.check/tests/integration/cves/outcome/no_fail.json

```diff
@@ -3,7 +3,7 @@
     "info": [
         [
             [
-                "CVE data downloaded from: NIST_URL"
+                "No CPE listed for: bazel_compdb"
```

Comment:
is this correct?

Reply:
hmm, not sure, good spot, ill look further

Reply:
so the removal is correct - it doesnt print that anymore in the info log (it has a debug log instead)

Reply:
and yep, its not an addition, its just the diff from removing the first log  - so i think it is correct

Reply:
i think i called it that because it wasnt a total failure - but the paths where this is the outcome have that `log.info` in the output

its kinda no_fail_but_no_anything_else_either